sdks/core/src/main/java/org/gbif/pipelines/core/interpreters/extension/MultimediaInterpreter.java

```diff
@@ -160,6 +160,8 @@ private static void parseAndSetType(Multimedia m, String v) {
       m.setType(MediaType.Sound.name());
     } else if (prFn.test("image", MediaType.StillImage)) {
       m.setType(MediaType.StillImage.name());
```

Comment:
are these changes intended to be included in this PR ?
I think we need to reference a separate issue.

Reply:
I made multimedia changes, so they mustn't be part of PR